scripts/docker-compose-config.bash

```diff
@@ -57,19 +57,20 @@ if docker compose version --short | grep --quiet "^2\." ; then
   if [[ -z "$version" ]]; then
     version="3.9"  # Default to 3.9 if version is not found in file
   fi
-
+# shellcheck disable=SC2002
   docker_command="\
-docker \
---log-level=ERROR \
-compose \
---env-file ${env_file}"
+set -o allexport && \
+export $(cat "${env_file}" | sort) && set +o allexport && \
+docker stack config"
```

Comment:
:hot:
that is new. what are the implications? and does it use the .env file as well?
do we need to copy that also in osparc-simcore? since this script comes from there. you totally remove docker-compose v1 as well right? that is good.

Reply:
this bash-code sources the `.env` file, which is how the docker guys expect you to use "docker stack config", I guess you should also use `docker stack config` in osparc-simcore. I actually copied the bash script from there originally, so you might get lucky and it is possible to just replace the one in `osparc-simcore` with this version